manifests/m/Minecodes/4zur3/1.1.1/Minecodes.4zur3.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Minecodes.4zur3
+PackageVersion: 1.1.1
```

```suggestion
PackageVersion: 1.1.1.0
```
This must match Control Panel.

You'll also need to change `manifests/m/Minecodes/4zur3/1.1.1` to `manifests/m/Minecodes/4zur3/1.1.1.0`